egs2/zh_openslr38/asr1/run_sslr.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
can you move either of them (current `run.sh` or this `run_sslr.sh`) under the local directory?
For example, 
- if you want to keep the `run_sslr.sh` in the main directory, you can rename run_sslr.sh as `run.sh` and move the current non SSLR `run.sh` to `local/run_nosslr.sh`
- adding the description of both `run.sh` and `local/run_nosslr.sh` to README.md

We want to keep one `run.sh` in the main recipe directory.

Reply:
Gotcha, will do!